src/main/resources/application.conf

```diff
@@ -39,9 +39,17 @@ notifications {
     prefix = "[Hercules]"
     channels = ["critical"]
     retry_interval = 60
-    # 0 = infinite retries
-    num_retries = 0  
-  }  
+    num_retries = 0
+  }
+  slack {
+    channels = ["progress"]
+    retry_interval = 60
+    num_retries = 2
+    slack_endpoint = "https://hooks.slack.com/services/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"
+    slack_channel = "testing-webhooks"
```

Comment:
I think we can use a more permanent name for the slack channel, e.g. "hercules-progress-uppsala" or something like that